2019-07-22 15:47:54,432 ERROR 21880 nodejs.ClusterClientNoResponseError: client no response in 1170442ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mac/Desktop/blog-web/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21880
hostname: MacBook-Pro-3.local

2019-07-22 20:32:06,043 ERROR 21880 nodejs.ClusterClientNoResponseError: client no response in 1908884ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mac/Desktop/blog-web/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21880
hostname: MacBook-Pro-3.local

2019-07-22 21:50:34,490 ERROR 21880 nodejs.ClusterClientNoResponseError: client no response in 4668536ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mac/Desktop/blog-web/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21880
hostname: MacBook-Pro-3.local

2019-07-23 00:37:51,259 ERROR 21880 nodejs.ClusterClientNoResponseError: client no response in 7216520ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mac/Desktop/blog-web/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21880
hostname: MacBook-Pro-3.local

2019-07-23 02:38:27,682 ERROR 21880 nodejs.ClusterClientNoResponseError: client no response in 7216419ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mac/Desktop/blog-web/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21880
hostname: MacBook-Pro-3.local

2019-07-23 04:39:23,225 ERROR 21880 nodejs.ClusterClientNoResponseError: client no response in 7215499ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mac/Desktop/blog-web/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21880
hostname: MacBook-Pro-3.local

2019-07-23 06:39:58,657 ERROR 21880 nodejs.ClusterClientNoResponseError: client no response in 7215434ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mac/Desktop/blog-web/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21880
hostname: MacBook-Pro-3.local

2019-07-23 08:40:54,072 ERROR 21880 nodejs.ClusterClientNoResponseError: client no response in 7215403ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mac/Desktop/blog-web/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21880
hostname: MacBook-Pro-3.local

2019-07-23 09:45:09,357 ERROR 21880 nodejs.ClusterClientNoResponseError: client no response in 3835280ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mac/Desktop/blog-web/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21880
hostname: MacBook-Pro-3.local

2019-07-23 12:38:46,500 ERROR 21880 nodejs.ClusterClientNoResponseError: client no response in 1755367ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mac/Desktop/blog-web/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21880
hostname: MacBook-Pro-3.local

2019-07-23 13:37:12,413 ERROR 21880 nodejs.ClusterClientNoResponseError: client no response in 1905610ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/mac/Desktop/blog-web/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 21880
hostname: MacBook-Pro-3.local

